pubtools/_quay/operator_pusher.py

```diff
@@ -420,30 +429,64 @@ def build_index_images(self):
         iib_results = {}
 
         for version, items in sorted(self.version_items_mapping.items()):
-            bundles = [self.public_bundle_ref(i) for i in items]
-            all_archs = [
-                i.metadata["arch"] if i.metadata["arch"] != "x86_64" else "amd64" for i in items
-            ]
-            archs = sorted(list(set(all_archs)))
-            signing_keys = sorted(list(set([item.claims_signing_key for item in items])))
+            is_hotfix = any([item.metadata.get("com.redhat.hotfix") for item in items])
+            is_advisory_source = all(
+                [re.match(r"^[A-Z0-9:\-]{4,40}$", item.origin) for item in items]
```

Comment:
This regex is not immediately parsable, can you add a comment on what it means?

Reply:
I took it from pushsource. Although I agree it's little strange